docker-compose.yml

```
@@ -0,0 +1,19 @@
+version: "3"
```

I get an error with this version saying that `target` is unrecognised. Had to update this to 3.8 to get it working:
```suggestion
version: "3.8"
```

---

Interesting - it worked for me. Updating regardless!